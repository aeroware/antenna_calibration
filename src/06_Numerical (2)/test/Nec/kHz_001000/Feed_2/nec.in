CM Testcube - V0 mesh
GW 1 1 0.000000 0.000000 0.500000 0.000000 0.000000 0.830000 0.002000
GW 1 1 0.000000 0.500000 0.000000 0.000000 0.830000 0.000000 0.002000
GW 1 1 0.000000 0.000000 0.830000 0.000000 0.000000 1.165000 0.002000
GW 1 1 0.000000 0.000000 1.165000 0.000000 0.000000 1.500000 0.002000
GW 1 1 0.000000 0.830000 0.000000 0.000000 1.165000 0.000000 0.002000
GW 1 1 0.000000 1.165000 0.000000 0.000000 1.500000 0.000000 0.002000
SP 0 2 -0.500000 -0.500000 -0.500000 0.000000 0.000000 -0.500000
SC 0 0 0.500000 -0.500000 -0.500000
SP 0 2 -0.500000 0.500000 -0.500000 0.000000 0.000000 -0.500000
SC 0 0 -0.500000 -0.500000 -0.500000
SP 0 2 0.500000 0.500000 -0.500000 0.000000 0.000000 -0.500000
SC 0 0 -0.500000 0.500000 -0.500000
SP 0 2 0.500000 -0.500000 -0.500000 0.000000 0.000000 -0.500000
SC 0 0 0.500000 0.500000 -0.500000
SP 0 2 0.500000 -0.500000 -0.500000 0.500000 0.000000 0.000000
SC 0 0 0.500000 -0.500000 0.500000
SP 0 2 0.500000 0.000000 0.000000 0.500000 -0.500000 -0.500000
SC 0 0 0.500000 0.500000 -0.500000
SP 0 2 0.500000 0.000000 0.000000 0.500000 0.500000 -0.500000
SC 0 0 0.500000 0.500000 0.500000
SP 0 2 0.500000 0.000000 0.000000 0.500000 0.500000 0.500000
SC 0 0 0.500000 -0.500000 0.500000
SP 0 2 0.000000 -0.500000 0.000000 -0.500000 -0.500000 -0.500000
SC 0 0 0.500000 -0.500000 -0.500000
SP 0 2 0.000000 -0.500000 0.000000 0.500000 -0.500000 -0.500000
SC 0 0 0.500000 -0.500000 0.500000
SP 0 2 0.000000 -0.500000 0.000000 0.500000 -0.500000 0.500000
SC 0 0 -0.500000 -0.500000 0.500000
SP 0 2 -0.500000 -0.500000 -0.500000 0.000000 -0.500000 0.000000
SC 0 0 -0.500000 -0.500000 0.500000
SP 0 2 -0.500000 0.000000 0.000000 -0.500000 0.500000 -0.500000
SC 0 0 -0.500000 -0.500000 -0.500000
SP 0 2 -0.500000 0.000000 0.000000 -0.500000 -0.500000 -0.500000
SC 0 0 -0.500000 -0.500000 0.500000
SP 0 2 -0.500000 0.000000 0.000000 -0.500000 -0.500000 0.500000
SC 0 0 -0.500000 0.500000 0.500000
SP 0 2 -0.500000 0.500000 -0.500000 -0.500000 0.000000 0.000000
SC 0 0 -0.500000 0.500000 0.500000
SP 0 0 -0.333333 0.000000 -0.333333 -0.000000 -90.000000 1.000000
SP 0 0 0.666667 0.000000 0.000000 90.000000 0.000000 1.000000
GE 0 0
FR 0 1 0 0 1000000.000000 0
EX 0 0 3 0 1.000000e-006 0 0
EX 0 0 5 0 1 0 0
XQ 0
EN