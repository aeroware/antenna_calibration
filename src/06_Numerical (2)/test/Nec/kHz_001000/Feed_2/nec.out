1



                                ***********************************************
                                *                                             *
                                *  NUMERICAL ELECTROMAGNETICS CODE (NEC-4.1)  *
                                *                                             *
                                ***********************************************




                      **********************************************************

                          Testcube - V0 mesh                                                           

                      **********************************************************





                                 - - - STRUCTURE SPECIFICATION - - -

                                     COORDINATES MUST BE INPUT IN
                                     METERS OR BE SCALED TO METERS
                                     BEFORE STRUCTURE INPUT IS ENDED


  WIRE                                                                               NO. OF    FIRST  LAST     TAG
  NO.        X1         Y1         Z1          X2         Y2         Z2      RADIUS   SEG.     SEG.   SEG.     NO.
     1    0.00000    0.00000    0.50000     0.00000    0.00000    0.83000    0.00200      1        1     1       1
     2    0.00000    0.50000    0.00000     0.00000    0.83000    0.00000    0.00200      1        2     2       1
     3    0.00000    0.00000    0.83000     0.00000    0.00000    1.16500    0.00200      1        3     3       1
     4    0.00000    0.00000    1.16500     0.00000    0.00000    1.50000    0.00200      1        4     4       1
     5    0.00000    0.83000    0.00000     0.00000    1.16500    0.00000    0.00200      1        5     5       1
     6    0.00000    1.16500    0.00000     0.00000    1.50000    0.00000    0.00200      1        6     6       1
     1T  -0.50000   -0.50000   -0.50000     0.00000    0.00000   -0.50000
          0.50000   -0.50000   -0.50000     0.00000    0.00000    0.00000
     2T  -0.50000    0.50000   -0.50000     0.00000    0.00000   -0.50000
         -0.50000   -0.50000   -0.50000     0.00000    0.00000    0.00000
     3T   0.50000    0.50000   -0.50000     0.00000    0.00000   -0.50000
         -0.50000    0.50000   -0.50000     0.00000    0.00000    0.00000
     4T   0.50000   -0.50000   -0.50000     0.00000    0.00000   -0.50000
          0.50000    0.50000   -0.50000     0.00000    0.00000    0.00000
     5T   0.50000   -0.50000   -0.50000     0.50000    0.00000    0.00000
          0.50000   -0.50000    0.50000     0.00000    0.00000    0.00000
     6T   0.50000    0.00000    0.00000     0.50000   -0.50000   -0.50000
          0.50000    0.50000   -0.50000     0.00000    0.00000    0.00000
     7T   0.50000    0.00000    0.00000     0.50000    0.50000   -0.50000
          0.50000    0.50000    0.50000     0.00000    0.00000    0.00000
     8T   0.50000    0.00000    0.00000     0.50000    0.50000    0.50000
          0.50000   -0.50000    0.50000     0.00000    0.00000    0.00000
     9T   0.00000   -0.50000    0.00000    -0.50000   -0.50000   -0.50000
          0.50000   -0.50000   -0.50000     0.00000    0.00000    0.00000
    10T   0.00000   -0.50000    0.00000     0.50000   -0.50000   -0.50000
          0.50000   -0.50000    0.50000     0.00000    0.00000    0.00000
    11T   0.00000   -0.50000    0.00000     0.50000   -0.50000    0.50000
         -0.50000   -0.50000    0.50000     0.00000    0.00000    0.00000
    12T  -0.50000   -0.50000   -0.50000     0.00000   -0.50000    0.00000
         -0.50000   -0.50000    0.50000     0.00000    0.00000    0.00000
    13T  -0.50000    0.00000    0.00000    -0.50000    0.50000   -0.50000
         -0.50000   -0.50000   -0.50000     0.00000    0.00000    0.00000
    14T  -0.50000    0.00000    0.00000    -0.50000   -0.50000   -0.50000
         -0.50000   -0.50000    0.50000     0.00000    0.00000    0.00000
    15T  -0.50000    0.00000    0.00000    -0.50000   -0.50000    0.50000
         -0.50000    0.50000    0.50000     0.00000    0.00000    0.00000
    16T  -0.50000    0.50000   -0.50000    -0.50000    0.00000    0.00000
         -0.50000    0.50000    0.50000     0.00000    0.00000    0.00000
    17P  -0.33333    0.00000   -0.33333    -0.00000  -90.00000    1.00000
    18P   0.66667    0.00000    0.00000    90.00000    0.00000    1.00000

   TOTAL SEGMENTS USED=    6     NO. SEG. IN A SYMMETRIC CELL=    6     SYMMETRY FLAG=  0
   TOTAL PATCHES USED=   18      NO. PATCHES IN A SYMMETRIC CELL=   18


         - MULTIPLE WIRE JUNCTIONS -
 JUNCTION    SEGMENTS  (- FOR END 1, + FOR END 2)
  NONE




                                 - - - - SEGMENTATION DATA - - - -

                                        COORDINATES IN METERS

                         I+ AND I- INDICATE THE SEGMENTS BEFORE AND AFTER I


  SEG.   COORDINATES OF SEG. CENTER     SEG.     ORIENTATION ANGLES    WIRE    CONNECTION DATA   TAG
  NO.       X         Y         Z       LENGTH     ALPHA     BETA      RADIUS    I-   I    I+    NO.
     1   0.00000   0.00000   0.66500   0.33000   90.00000   0.00000   0.00200     0    1    3      1
     2   0.00000   0.66500   0.00000   0.33000    0.00000  90.00000   0.00200     0    2    5      1
     3   0.00000   0.00000   0.99750   0.33500   90.00000   0.00000   0.00200     1    3    4      1
     4   0.00000   0.00000   1.33250   0.33500   90.00000   0.00000   0.00200     3    4    0      1
     5   0.00000   0.99750   0.00000   0.33500    0.00000  90.00000   0.00200     2    5    6      1
     6   0.00000   1.33250   0.00000   0.33500    0.00000  90.00000   0.00200     5    6    0      1




                                            - - - SURFACE PATCH DATA - - -

                                                 COORDINATES IN METERS

 PATCH     COORD. OF PATCH CENTER       UNIT NORMAL VECTOR      PATCH            COMPONENTS OF UNIT TANGENT VECTORS
  NO.      X         Y         Z         X       Y       Z       AREA       X1      Y1      Z1       X2      Y2      Z2
    1   0.00000  -0.33333  -0.50000   0.0000  0.0000 -1.0000   0.25000   0.7071  0.7071  0.0000   0.7071 -0.7071  0.0000
    2  -0.33333   0.00000  -0.50000   0.0000 -0.0000 -1.0000   0.25000   0.7071 -0.7071  0.0000  -0.7071 -0.7071  0.0000
    3   0.00000   0.33333  -0.50000  -0.0000  0.0000 -1.0000   0.25000  -0.7071 -0.7071  0.0000  -0.7071  0.7071  0.0000
    4   0.33333   0.00000  -0.50000   0.0000  0.0000 -1.0000   0.25000  -0.7071  0.7071  0.0000   0.7071  0.7071  0.0000
    5   0.50000  -0.33333   0.00000   1.0000  0.0000 -0.0000   0.25000   0.0000  0.7071  0.7071   0.0000 -0.7071  0.7071
    6   0.50000   0.00000  -0.33333   1.0000  0.0000  0.0000   0.25000   0.0000 -0.7071 -0.7071   0.0000  0.7071 -0.7071
    7   0.50000   0.33333   0.00000   1.0000 -0.0000  0.0000   0.25000   0.0000  0.7071 -0.7071   0.0000  0.7071  0.7071
    8   0.50000   0.00000   0.33333   1.0000  0.0000 -0.0000   0.25000   0.0000  0.7071  0.7071   0.0000 -0.7071  0.7071
    9   0.00000  -0.50000  -0.33333   0.0000 -1.0000 -0.0000   0.25000  -0.7071  0.0000 -0.7071   0.7071  0.0000 -0.7071
   10   0.33333  -0.50000   0.00000   0.0000 -1.0000  0.0000   0.25000   0.7071  0.0000 -0.7071   0.7071  0.0000  0.7071
   11   0.00000  -0.50000   0.33333   0.0000 -1.0000  0.0000   0.25000   0.7071  0.0000  0.7071  -0.7071  0.0000  0.7071
   12  -0.33333  -0.50000   0.00000   0.0000 -1.0000  0.0000   0.25000   0.7071  0.0000  0.7071  -0.7071  0.0000  0.7071
   13  -0.50000   0.00000  -0.33333  -1.0000  0.0000 -0.0000   0.25000   0.0000  0.7071 -0.7071   0.0000 -0.7071 -0.7071
   14  -0.50000  -0.33333   0.00000  -1.0000 -0.0000  0.0000   0.25000   0.0000 -0.7071 -0.7071   0.0000 -0.7071  0.7071
   15  -0.50000   0.00000   0.33333  -1.0000  0.0000  0.0000   0.25000   0.0000 -0.7071  0.7071   0.0000  0.7071  0.7071
   16  -0.50000   0.33333   0.00000  -1.0000  0.0000  0.0000   0.25000   0.0000 -0.7071  0.7071   0.0000  0.7071  0.7071
   17  -0.33333   0.00000  -0.33333  -0.0000 -1.0000  0.0000   1.00000   1.0000 -0.0000  0.0000  -0.0000  0.0000  1.0000
   18   0.66667   0.00000   0.00000   0.0000 -0.0000  1.0000   1.00000   1.0000  0.0000  0.0000  -0.0000  1.0000  0.0000




 ***** INPUT LINE  1  FR   0    1    0    0  1.00000E+06  0.00000E+00  0.00000E+00  0.00000E+00  0.00000E+00  0.00000E+00
 ***** INPUT LINE  2  EX   0    0    3    0  1.00000E-06  0.00000E+00  0.00000E+00  0.00000E+00  0.00000E+00  0.00000E+00
 ***** INPUT LINE  3  EX   0    0    5    0  1.00000E+00  0.00000E+00  0.00000E+00  0.00000E+00  0.00000E+00  0.00000E+00
 ***** INPUT LINE  4  XQ   0    0    0    0  0.00000E+00  0.00000E+00  0.00000E+00  0.00000E+00  0.00000E+00  0.00000E+00




                                 - - - - - - FREQUENCY - - - - - -

                                    FREQUENCY= 1.0000E+06 MHZ
                                    WAVELENGTH= 2.9980E-04 METERS



                                  - - - ANTENNA ENVIRONMENT - - -

                                            FREE SPACE



                               - - - STRUCTURE IMPEDANCE LOADING - - -

                                   THIS STRUCTURE IS NOT LOADED



                                - - - MATRIX TIMING - - -

                        FILL=    1.062 SEC.,  FACTOR=    0.016 SEC.



                                          - - - ANTENNA INPUT PARAMETERS - - -

   TAG   SEG.    VOLTAGE (VOLTS)         CURRENT (AMPS)         IMPEDANCE (OHMS)        ADMITTANCE (MHOS)      POWER
   NO.   NO.    REAL        IMAG.       REAL        IMAG.       REAL        IMAG.       REAL        IMAG.     (WATTS)
     1     3 1.00000E-06 0.00000E+00 1.82944E-24 3.98109E-24 9.53035E+16-2.07392E+17 1.82944E-18 3.98109E-18 9.14722E-31
     1     5 1.00000E+00 0.00000E+00 3.68041E-18-5.94407E-19 2.64802E+17 4.27669E+16 3.68041E-18-5.94407E-19 1.84021E-18



                             - - - CURRENTS AND LOCATION - - -

                     LENGTHS NORMALIZED BY WAVELENGTH (OR 2.*PI/CABS(K))

  SEG.  TAG    COORD. OF SEG. CENTER     SEG.            - - - CURRENT (AMPS) - - -
  NO.   NO.     X        Y        Z      LENGTH     REAL        IMAG.       MAG.        PHASE
     1    1   0.0000   0.00002218.1454*********   1.7899E-24  2.6220E-24  3.1747E-24   55.681
     2    1   0.00002218.1454   0.0000*********   2.9550E-18 -1.0653E-18  3.1411E-18  -19.824
     3    1   0.0000   0.00003327.2181*********   1.8294E-24  3.9811E-24  4.3813E-24   65.320
     4    1   0.0000   0.00004444.6298*********  -4.7146E-25 -5.1367E-25  6.9723E-25 -132.547
     5    1   0.00003327.2181   0.0000*********   3.6804E-18 -5.9441E-19  3.7281E-18   -9.174
     6    1   0.00004444.6298   0.0000*********  -1.9600E-18  1.0777E-18  2.2368E-18  151.197




                                         - - - - SURFACE PATCH CURRENTS - - - -

                                                  DISTANCES IN WAVELENGTHS (2.*PI/CABS(K))
                                                  CURRENT IN AMPS/METER

                            - - SURFACE COMPONENTS - -                   - - - RECTANGULAR COMPONENTS - - -
      PATCH CENTER      TANGENT VECTOR 1   TANGENT VECTOR 2           X                   Y                   Z
     X      Y      Z     MAG.       PHASE   MAG.       PHASE    REAL      IMAG.     REAL      IMAG.     REAL      IMAG. 
    1
   0.000************** 3.7527E-21   52.50 1.7140E-20   -4.10  1.37E-20  1.24E-21 -1.05E-20  2.97E-21  0.00E+00  0.00E+00
    2
 *******  0.000******* 3.4219E-20  -79.32 1.3743E-20  130.79  1.08E-20 -3.11E-20  1.86E-21  1.64E-20  0.00E+00  0.00E+00
    3
   0.000************** 4.2739E-20 -155.12 1.1755E-20 -172.86  3.57E-20  1.37E-20  1.92E-20  1.17E-20  0.00E+00 -0.00E+00
    4
 *******  0.000******* 1.4551E-20  136.05 4.9610E-20  -46.96  3.14E-20 -3.28E-20  1.65E-20 -1.85E-20  0.00E+00  0.00E+00
    5
 **************  0.000 6.5551E-20  145.09 4.0892E-20   11.02  0.00E+00  0.00E+00 -6.64E-20  2.10E-20 -9.63E-21  3.21E-20
    6
 *******  0.000******* 2.7474E-20  -82.03 1.2643E-20  -34.93  0.00E+00  0.00E+00  4.63E-21  1.41E-20 -1.00E-20  2.44E-20
    7
 **************  0.000 1.2321E-19  -64.24 1.0883E-19  -74.67  0.00E+00  0.00E+00  5.82E-20 -1.53E-19 -1.75E-20  4.25E-21
    8
 *******  0.000******* 7.9038E-20 -109.07 5.9180E-20  -84.45  0.00E+00  0.00E+00 -2.23E-20 -1.12E-20 -1.42E-20 -9.45E-20
    9
   0.000************** 2.5042E-20  -58.30 2.2335E-20   51.10  6.14E-22  2.74E-20  0.00E+00  0.00E+00 -1.92E-20  2.78E-21
   10
 **************  0.000 5.1264E-20 -102.29 4.1666E-20 -110.12 -1.78E-20 -6.31E-20  0.00E+00 -0.00E+00 -2.42E-21  7.75E-21
   11
   0.000************** 7.4307E-21 -157.03 4.7339E-20   75.37 -1.33E-20 -3.44E-20  0.00E+00  0.00E+00  3.61E-21  3.03E-20
   12
 **************  0.000 5.2849E-20 -112.73 2.8339E-20   62.77 -2.36E-20 -5.23E-20  0.00E+00  0.00E+00 -5.27E-21 -1.67E-20
   13
 *******  0.000******* 3.8839E-20  -26.17 8.4896E-21   72.21  0.00E+00  0.00E+00  2.28E-20 -1.78E-20 -2.65E-20  6.40E-21
   14
 **************  0.000 3.8912E-20  138.53 5.8667E-20  160.76 -0.00E+00  0.00E+00  5.98E-20 -3.19E-20 -1.86E-20 -4.55E-21
   15
 *******  0.000******* 3.8088E-21  164.31 6.5857E-20   50.06  0.00E+00  0.00E+00  3.25E-20  3.50E-20  2.73E-20  3.64E-20
   16
 **************  0.000 1.2243E-19  -90.13 8.3318E-20  122.04  0.00E+00  0.00E+00 -3.11E-20  1.37E-19 -3.15E-20 -3.66E-20
   17
 *******  0.000******* 7.7246E-21 -144.93 6.6011E-21   72.00 -6.32E-21 -4.44E-21  3.23E-32  2.26E-32  2.04E-21  6.28E-21
   18
 *******  0.000  0.000 7.5326E-21 -105.26 1.2156E-20 -139.71 -1.98E-21 -7.27E-21 -9.27E-21 -7.86E-21  0.00E+00 -0.00E+00



                                        - - - POWER BUDGET - - -

                                           INPUT POWER   = 1.8402E-18 WATTS
                                           RADIATED POWER= 1.8402E-18 WATTS
                                           WIRE LOSS     = 0.0000E+00 WATTS
                                           EFFICIENCY    = 100.00 PERCENT




 ***** INPUT LINE  5  EN   0    0    0    0  0.00000E+00  0.00000E+00  0.00000E+00  0.00000E+00  0.00000E+00  0.00000E+00

 RUN TIME =     1.078
